packages/arb-token-bridge-ui/src/util/deposits/helpers.ts

```diff
@@ -60,14 +60,25 @@ export const updateAdditionalDepositData = async ({
 
   const { isClassic } = depositTx // isClassic is known before-hand from subgraphs
 
-  const isEthDeposit = depositTx.assetType === AssetType.ETH
+  const isNativeTokenTransferToSameAddress =
+    depositTx.assetType === AssetType.ETH &&
+    // We cannot just check for tx.sender.toLowerCase() === tx.destination.toLowerCase()
+    // because if destination or sender is undefined, this will be assigned false
+    // but if that's the case we should treat it as transfer to the same address
+    !(
+      depositTx.sender &&
+      depositTx.destination &&
+      depositTx.sender.toLowerCase() !== depositTx.destination.toLowerCase()
+    )
 
   const { parentToChildMsg } =
     await getParentToChildMessageDataFromParentTxHash({
       depositTxId: depositTx.txID,
       parentProvider,
       childProvider,
-      isEthDeposit,
+      // we use `depositTo` from arbitrum-sdk to send native token to a different destination address
+      // it uses retryables so technically it's not ETH deposit
+      isEthDeposit: isNativeTokenTransferToSameAddress,
```

Comment:
We can replace `isEthDeposit` with `isRetryableDeposit` and invert it's usage in the internal function.